[ERROR] [1612488937.199917748]: An error has occurred during frame callback: map::at

An error has occurred during frame callback: 
[ERROR] [1612488937.199917748]: An error has occurred during frame callback: map::at

Hi there,

Specs: Running Ubuntu 18.04 LTS on an desktop, Camera: L515, USB3 connection, SDK: 2.41.0.

I am trying to get a point cloud with the following launch command. roslaunch realsense2_camera rs_camera.launch
The point cloud and camera image show up fine and I don't visually see any problem other than the following:

I seem to consistently get the error message shown in the terminal: 
[ERROR] [1612488936.949575351]: An error has occurred during frame callback: map::at
[ERROR] [1612488937.050368488]: An error has occurred during frame callback: map::at
[ERROR] [1612488937.134904899]: An error has occurred during frame callback: map::at
...

I have tried installing the SDK with Debian Package as well as from source and then following this.

Any and all help would be greatly appreciated, thanks in advance.

--- a/realsense2_camera/src/base_realsense_node.cpp
+++ b/realsense2_camera/src/base_realsense_node.cpp
@@ -953,7 +953,7 @@ void BaseRealSenseNode::publishAlignedDepthToOthers(rs2::frameset frames, const 
         auto frame = (*it);
         auto stream_type = frame.get_profile().stream_type();
 
-        if (RS2_STREAM_DEPTH == stream_type)
+        if (RS2_STREAM_DEPTH == stream_type || RS2_STREAM_CONFIDENCE == stream_type)
             continue;
 
         auto stream_index = frame.get_profile().stream_index();

